specification/cognitiveservices/data-plane/CustomVision/Prediction/stable/v3.0/Prediction.json.orig

```diff
@@ -0,0 +1,1160 @@
+﻿{
```

Comment:
I guess this got checked in by accident?

Reply:
Review has been requested - not complete yet.
Yes, thanks I removed the .orig files..

Reply:
Review has been signed-off